### DISCUSSION TITLE
My First Htwoo Webpart

### DISCUSSION THREAD

**Discussion by robpearmain** [n8design/htwoo#7] [Show and tell]
I removed all references to Office UI Fabric, and installed the library, so easy!

<img width="295" alt="Screenshot 2021-04-28 at 17 28 31" src="https://user-images.githubusercontent.com/299240/116439303-4fce5a00-a847-11eb-96fc-39a1fe21183b.png">

**Comment by StfBauer** [comment]
Glad you like it - Thank you very much for your feedback.
